Topshelf.Linux/packages.config

```diff
@@ -15,5 +14,6 @@
 // specific language governing permissions and limitations under the License.
 -->
 <packages>
-	<package id="Topshelf" version="3.2.0" targetFramework="net40" />
+  <package id="Mono.Posix" version="4.0.0.0" targetFramework="net40" />
```

Comment:
Since there is a conditional check using MonoHelper.RunningOnLinux I don't think there will ever be an issue of working on both Windows and Mono/Linux since on Windows the code is never JIT'd. It should be lazy and only loaded once executed. That will only happen on Linux and there its ok to call the Unix Signals. Do you agree with this statement? That is how I understand things should work but I might be wrong. FYI..I did test my application on both Windows and Ubuntu and both ran without issue.

Looking at trying to bring in the functionality required for Unix signals might be more complicated and brittle. I would need to dig deeper but at first glance a decent amount of code will need to be ported over into MonoHelper.